Towner & Combat Pilgrim Jobs Re-Skilling + Athletic & Mining Skill Matters!!!!

Stutternov opened the pull request:
## About The Pull Request

I should have divided this PR in 2 but I did the balancing based off the skill changes themselves.

**To briefly describe what this does:**
- **Ports over athletics being an added modifier** to your max stamina cap; also allows for athletic skill leveling from use. (Athletics, currently, only impacts knockback. This also makes it impact max-stamina.)
- **Ports over the reworked pickaxe skill from Blackstone**; basically making pickaxes not comedically slow and instead just rewards you based off their usage. Mining skill now is far more helpful, can be leveled, and are impacted based on toolspeed using pick-type attack.
-  **Adds two new towner roles**, one being _Noble_, which is a changed version of Blackstone's Noble pilgrim, and the rare _Merchant_ pilgrim, which is a basically peddler of random goods which determines his skill. Similar to the merchant and noble pilgrim combined, albeit no access to merchant stuff.
- **Reworked (most) towner-pilgrim roles**; basically just changed woodcutter and carpenter to be a bit unique from one another and not as CBT to play, rewarding crafting. Also changed some other ones like miner, giving him better skills, etc. (Also peasants are stinky and don't mine stank no more) A good example here is: Woodcutter, prior, just being low intel worse carpenter. Now, instead, they aren't as low of intel but carpenter simply gets more boons to crafting.
- **Reworked (some) combat-pilgrim roles**; basically took away some things like medium armor training from rogue but made their mood no longer impacted by literally murdering/delimbing people, gave minor buffs to some classes that felt left behind, and did some other small misc changes; such as molding skills/stats better to fit the role. (Devs suggested to remove medium armor training from Barb and Amazon due to their strength stat, done.)
- **Misc role changes.** Mason got a mild few changes, such as basic skills added to him to help him out incase no pilgrims are around or extra labor. Athletics added to more classes, as it's actually USEFUL now due to the skill change. Etc.

I do plan to add stuff to clergy, monk, etc but first I need to go fix our faith system to actually make patron selection work. Which - is maybe my next project.

The **goal** of this PR is just to handle some forgotten stats and redo some combat pilgrims to lean into specific playstyles, while encouraging towner-pilgrims to be a bit less combat focused but be rewarded with better non-combat-related stats. People have non-stop complained about this, including myself about 1-2 classes, for over a week - so, here, I finally did what I said I'd do if I got around to it.  _This took me longer than it should have.._ 

## Why It's Good For The Game

This has been something talked about by devs and players alike for awhile with tons of different opinions of how to fix it; this is basically just an amalgamation of suggestions I've heard and talked to others about.

**That said I do genuinely want feedback on the skills, traits, etc.** I'm winging it here partially as everyone plays these roles differently and some ideas may be outright better than my own or previously stated ones. 

Also - I feel the two skill changes will reward certain classes and combat-styles more, as well as fix some of our issues we've had with certain classes.

## Pre-Merge Checklist
<!-- Don't bother filling these in while creating your Pull Request, just click the checkboxes after the Pull Request is opened and you are redirected to the page. -->
- [X] You tested this on a local server.
- [X] This code did not runtime during testing.
- [X] You documented all of your changes.
<!-- Neither the compiler nor workflow checks are perfect at detecting runtimes and errors. It is important to test your code/feature/fix locally. -->

BadAtThisGame302 commented on code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/noble.dm:
```diff
@@ -0,0 +1,51 @@
+	tutorial = "You are a noble, from some place far away, trying to visit the nobility of the area \
+	you are encouraged to make your own story. But with wealth, come the poor, ready to pilfer you \
+	of your hard earned (inherited) Mammons, so tread lightly unless you want to meet a grizzly end"
+	allowed_sexes = list("male", "female")
```
We only use (MALE, FEMALE) in this house.

Stutternov replied:
Amended; forgot we use the straight defines unlike Blackstone.

Stutternov commented:
I can see now while I was working on this other updates got merged, i.e Ranger losing medium armor and Rogue stat-rework; I've tried to amend for the current approved reworked stuff. 

Values should be better now.

Schmorgisborg commented on code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/woodcutter.dm:
```diff
@@ -37,6 +38,6 @@
 	backpack_contents = list(/obj/item/flint = 1)
-	H.change_stat("strength", 1)
-	H.change_stat("intelligence", -2)
+	H.change_stat("strength", 3)
```
That's a very high amount of strength, and the intelligence debuff is likely to balance out the rolls.

Stutternov replied:
This is, currently at least, the same strength bonus barbarian gets. Though, Barbarian gets a bunch of combat skills with it and actually an intelligence increase.

I could restore it to only 1 strength but this mainly was to put it apart from carpenter. (The -2 intel was annoying as it actually impacted their crafting, despite having better crafting skill.)

ThePainkiller replied:
Man I didn't even give Grenzel mercs 3 STR (on testing, 2 was fine to roll 12 STR min for the Zwei), this feels like a bit too much for what they do, a simple woodcutter stronger than a merc that kills for money?

Stutternov replied:
Amended to Painkiller's suggestion given.

Schmorgisborg commented on code/_onclick/item_attack.dm:
```diff
@@ -271,15 +271,20 @@
-					dullfactor = 1.5
+					var/mob/living/miner = user
+					var/mineskill = miner.mind.get_skill_level(/datum/skill/labor/mining)
+					dullfactor = 1.5 * (mineskill * 0.1)
```
Would someone with no mining skill have 0 dullfactor?

Stutternov replied:
I believe it miiight have that side-effect, I'll have to test it out, I do know how horrid it is to mine without mining skill currently (1.3 slower, so 30%) but I'll double check when I do testing later.

Stutternov replied:
Fixed it to the following calculation:

dullfactor = 1.5 - (mineskill * 0.1)

Works properly on testing now!

Schmorgisborg commented:
This looks really good- you've done incredible work on the new roles, lots of little tweaks that needed to be addressed.

BadAtThisGame302 commented on code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/carpenter.dm:
```diff
@@ -29,7 +29,9 @@
 		H.mind.adjust_skillrank(/datum/skill/misc/climbing, 2, TRUE)
-		H.mind.adjust_skillrank(/datum/skill/misc/reading, 1, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/reading, 2, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 1, TRUE)
```
Carpenter medicine?

ThePainkiller replied:
Same, why would a carpenter know how to do basic sew wounds...?

Stutternov replied:
Removed from carpenter.

BadAtThisGame302 commented on code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/rare/merchant.dm:
```diff
@@ -0,0 +1,63 @@
+	var/merchtype = pickweight(list("FOOD" = 4, "HEAL" = 2, "SILK" = 1, "GEMS" = 1))
+	switch(merchtype)
+		if("FOOD")		// Travelling food peddler
+			backpack_contents = list()
```
Forgot to add the food or just thinking of what to add?

Stutternov replied:
I'm a bit stumped on what to give them and forgot to put a placeholder set in; feel free to suggest some!

BadAtThisGame302 replied:
> I'm a bit stumped on what to give them and forgot to put a placeholder set in; feel free to suggest some!

Salumoi, Coppiette, Cheese, Saltfish, Crackers (All of these don't rot, except cheese which turns to age once rot timer passes.) Could also give him a bottle of Valorian Red or White too.

Stutternov replied:
Added!

ThePainkiller commented on code/_onclick/item_attack.dm:
```diff
@@ -271,15 +271,20 @@
-					dullfactor = 1.5
+					var/mob/living/miner = user
+					var/mineskill = miner.mind.get_skill_level(/datum/skill/labor/mining)
+					dullfactor = 1.5 * (mineskill * 0.1)
```
```suggestion
					dullfactor = 1.6 - (mineskill * 0.1)
```
This way:
Untrained = 1.6
Weak = 1.4
Skilled = 1.3
Expert = 1.2
Master = 1.1
Legendary = 1

ThePainkiller commented on code/modules/jobs/job_types/roguetown/adventurer/types/combat/amazon.dm:
```diff
@@ -36,14 +37,14 @@
+		H.mind.adjust_skillrank(/datum/skill/combat/polearms, 3, TRUE)
+	H.change_stat("strength", 2)
+	H.change_stat("intelligence", -2)
+	H.change_stat("perception", 2)
```
Honestly less perception or directly no perception at all for barbarians is fine, getting 3 STR wouldn't be far fetched if you remove this stat entirely. Unga bunga, after all.

ThePainkiller commented on code/modules/jobs/job_types/roguetown/adventurer/types/combat/barbarian.dm:
```diff
@@ -49,10 +50,10 @@
 			beltr = /obj/item/rogueweapon/mace/woodclub
 			H.mind.adjust_skillrank(/datum/skill/combat/axesmaces, 1, TRUE)
-	H.change_stat("strength", 3)
+	H.change_stat("strength", 2)
```
Same as above tbh, considering they're dumb as a brick with heavy INT penalties and unable to craft, they can have 3 STR, it fits them

Stutternov replied:
Amended to suggestion given.

ThePainkiller commented on code/modules/jobs/job_types/roguetown/adventurer/types/combat/rogue.dm:
```diff
@@ -16,15 +16,17 @@
 	..()
 	shoes = /obj/item/clothing/shoes/roguetown/boots
 	H.mind.adjust_skillrank(/datum/skill/combat/knives, 3, TRUE)
+	H.mind.adjust_skillrank(/datum/skill/combat/crossbows, 2, TRUE)
```
```suggestion
	H.mind.adjust_skillrank(/datum/skill/combat/crossbows, 3, TRUE)
```
Insanely punishing for their literal primary weapon, please don't. They are already heavily gimped by -1 STR on draw time.

ThePainkiller commented on code/modules/jobs/job_types/roguetown/adventurer/types/combat/rogue.dm:
```diff
@@ -16,15 +16,17 @@
 	shoes = /obj/item/clothing/shoes/roguetown/boots
 	H.mind.adjust_skillrank(/datum/skill/combat/knives, 3, TRUE)
+	H.mind.adjust_skillrank(/datum/skill/combat/crossbows, 2, TRUE)
+	H.mind.adjust_skillrank(/datum/skill/combat/bows, 3, TRUE)
```
```suggestion
	H.mind.adjust_skillrank(/datum/skill/combat/bows, 2, TRUE)
```

ThePainkiller commented on code/modules/jobs/job_types/roguetown/adventurer/types/combat/sorceress.dm:
```diff
@@ -26,6 +26,8 @@
 		H.mind.adjust_skillrank(/datum/skill/misc/reading, 4, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/magic/arcane, 3, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/craft/cooking, 1, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/craft/alchemy, 1, TRUE)
```
```suggestion
		H.mind.adjust_skillrank(/datum/skill/craft/alchemy, 2, TRUE)
```

ThePainkiller commented on code/modules/jobs/job_types/roguetown/adventurer/types/combat/sorceress.dm:
```diff
@@ -26,6 +26,8 @@
 		H.mind.adjust_skillrank(/datum/skill/magic/arcane, 3, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/craft/cooking, 1, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/craft/alchemy, 1, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 2, TRUE)
```
```suggestion
		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 1, TRUE)
```
These two skills should be the other way around since mages should have some unspecialized knowledge of potionmaking, but not have anything beyond basic anatomy knowledge.

ThePainkiller commented on code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/miner.dm:
```diff
@@ -31,12 +31,17 @@
 		H.mind.adjust_skillrank(/datum/skill/misc/swimming, 1, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/climbing, 2, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 1, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/athletics, pick(3,3,4), TRUE)
```
```suggestion
		H.mind.adjust_skillrank(/datum/skill/misc/athletics, 3, TRUE)
```
This one in particular will require some testing, as maces are high on stamina cost and Miners having at least a minimal skill on them would make these dudes powergamed for lowered mace miss penalties and have too much of a stam pool, but at the same time it's also **required** by their jobs because mining takes a **fuckton** of stamina....
4 might be a bit too high. I'd say to leave it at 3 for now and then we can fine-tune from there.

ThePainkiller commented on code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/monk.dm:
```diff
@@ -33,6 +33,7 @@
 		H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 4, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/sewing, 2, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/medicine, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/athletics, pick(1,1,2), TRUE)
```
```suggestion
		H.mind.adjust_skillrank(/datum/skill/misc/athletics, pick(2,2,3), TRUE)
```
Honestly considering they're meant to use a quarterstaff or throw hands, sounds fair that they'd be fairly good at cardio.

ThePainkiller commented on code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/rare/Lminer.dm:
```diff
@@ -28,8 +28,10 @@
 		H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 3, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 3, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/craft/crafting, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/labor/mining, 5, TRUE)
```
```suggestion
		H.mind.adjust_skillrank(/datum/skill/labor/mining, 6, TRUE)
```
Legendary, rare, they deserve to be masters of it already. Diggy Diggy hole.

ThePainkiller commented on code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/rare/Lminer.dm:
```diff
@@ -28,8 +28,10 @@
+		H.mind.adjust_skillrank(/datum/skill/labor/mining, 5, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/swimming, 3, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/misc/climbing, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/misc/athletics, pick(3,3,4), TRUE)
```
Balanced, but my suggestion is to lower this skill (and -1 endurance) if they're of old age.

ThePainkiller commented on code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/rare/merchant.dm:
```diff
@@ -0,0 +1,63 @@
+	shoes = /obj/item/clothing/shoes/roguetown/boots
+	belt = /obj/item/storage/belt/rogue/leather/black
+	beltr = /obj/item/flashlight/flare/torch/lantern
+	backl = /obj/item/storage/backpack/rogue/satchel
```
```suggestion
	backl = /obj/item/storage/backpack/rogue/backpack
```
Better defined first and then the populate contents, also you defined backl twice lol

ThePainkiller commented on code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/rare/merchant.dm:
```diff
@@ -0,0 +1,63 @@
+		shirt = /obj/item/clothing/suit/roguetown/shirt/undershirt/green
+		cloak = /obj/item/clothing/cloak/half
+		head = /obj/item/clothing/head/roguetown/fancyhat
+	backl = /obj/item/storage/backpack/rogue/backpack
```
```suggestion
```
Begone duplicate line.

ThePainkiller commented on code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/rare/merchant.dm:
```diff
@@ -0,0 +1,63 @@
+	backl = /obj/item/storage/backpack/rogue/backpack
+
+	//For how we decide what kind of merchant they are.
+	var/merchtype = pickweight(list("FOOD" = 4, "HEAL" = 2, "SILK" = 1, "GEMS" = 1))
```
You're welcome ;)

ThePainkiller commented on code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/woodcutter.dm:
```diff
@@ -37,6 +38,6 @@
 	backpack_contents = list(/obj/item/flint = 1)
-	H.change_stat("strength", 1)
-	H.change_stat("intelligence", -2)
+	H.change_stat("strength", 3)
```
```suggestion
	H.change_stat("strength", 2)
```

ThePainkiller commented on code/modules/jobs/job_types/roguetown/other/merc_classes/underdweller.dm:
```diff
@@ -26,7 +26,7 @@
         H.mind.adjust_skillrank(/datum/skill/combat/knives, 2, TRUE)
         H.mind.adjust_skillrank(/datum/skill/combat/axesmaces, 4, TRUE)
-        H.mind.adjust_skillrank(/datum/skill/mining, 3, TRUE)
+        H.mind.adjust_skillrank(/datum/skill/labor/mining, 3, TRUE)
```
Don't forget to give Underdweller the athletics skill too.

RoboticVodka replied:
Believe all mercs are currently lacking athletics not just Underdweller.

ThePainkiller commented on code/modules/mob/living/carbon/rogfatstam.dm:
```diff
@@ -14,14 +14,19 @@
 	if(HAS_TRAIT(src, TRAIT_NOFATSTAM))
 		return TRUE
+	if(m_intent == MOVE_INTENT_RUN)
+		mind.adjust_experience(/datum/skill/misc/athletics, (STAINT*0.02))
```
Are you sure this will work as intended? Wouldn't it be better on `rogfat_add` so that it procs every time fatigue is deducted from a character?

ThePainkiller replied:
Still asking for clarification on this one, have you tested running for 1 tile and checking your athletics exp gain on /mind/?

Stutternov replied:
This is the way it is coded on Blackstone, now that PR is updated going to check. Trying to think how I'd - really test this better, but lets see.

Stutternov replied:
![image](https://github.com/Darkrp-community/OpenKeep/assets/47883419/b202a347-7ef3-49a3-9e1e-98de8d2e8c09)
Seems to add itself properly; only thing I see is it's giving me roughly .26 EXP every tile traveled, give or take.

ThePainkiller replied:
Balanced and based.

BadAtThisGame302 commented on code/modules/jobs/job_types/roguetown/adventurer/types/pilgrim/fisher.dm:
```diff
@@ -25,6 +25,7 @@
 		H.mind.adjust_skillrank(/datum/skill/craft/crafting, pick(1,2), TRUE)
```
...gifts of Dendor." Ooohh heeellll naaaah change that to Abyssor or I'll make a PR and conflict yours rn.

ThePainkiller replied:
That was a description I added on #117 , what's wrong with dendor's gifts?

BadAtThisGame302 replied:
> That was a description I added on #117 , what's wrong with dendor's gifts?

It's the sea, Abyssor is the God of the sea.
![image](https://github.com/Darkrp-community/OpenKeep/assets/79063506/b1091f56-61c8-4578-b76b-7954ff4cb44c)

ThePainkiller replied:
I thought fishes, being "beasts", was from Dendor, I never saw the description of giving primordial men **food** and water, whoops

Stutternov replied:
Fixing it to Abyssor; not really my change but fuck it, why not fix it.

Stutternov commented:
Tonight I will get around to handling the suggested changes to get this merge-ready.

Stutternov commented:
Right, handled fixing the mining skill issue (Bad calculation, it's working now!) + added all the suggested comments. Should be ready for a review again, fully tested it (mostly the skills) and it's working.

ThePainkiller commented on code/_onclick/item_attack.dm:
```diff
@@ -276,7 +276,7 @@
 					var/mob/living/miner = user
 					var/mineskill = miner.mind.get_skill_level(/datum/skill/labor/mining)
-					dullfactor = 1.6 - (mineskill * 0.1)
+					dullfactor = 1.5 - (mineskill * 0.1)
```
So Legendary will have 0.9 as dullfactor?

Stutternov replied:
Shit, I thought lowering it might make it better but forgot we go up to 6 skill not 5.
Fixing it back to 1.6, I see why 1.6 is the default now.

carbous commented:
Right so my feedback on that one is that, first off about the noble one, they're kind of lame, they don't got much skills, stats and they don't spawn with that much money. they probably could use something more. second is about amazons, so the issue with the medium armor trait removal is that they can spawn with a medium armor, second making them weaker and dumber than before is fine but they could use con like barbarian if they're not supposed to wear armor.

Stutternov commented:
> Right so my feedback on that one is that, first off about the noble one, they're kind of lame, they don't got much skills, stats and they don't spawn with that much money. they probably could use something more. second is about amazons, so the issue with the medium armor trait removal is that they can spawn with a medium armor, second making them weaker and dumber than before is fine but they could use con like barbarian if they're not supposed to wear armor.

Amazon don't spawn with medium armor, the Chainkini is actually ARMOR_CLASS_LIGHT, funnily. The other 2 armors they can spawn with are both light armors as well.

I'll take a look at Noble, this is mainly supposed to be a relatively 'roleplay' based class (Idk why people exactly liked it), but I'll see what I can shove to them to make them at least a tad better.

Zoeycore commented:
**Warrior Nerfening, and Power Fantasy**

Nerfing the Warriors base skills goes against the grain of the Classes power fantasy, making it a lot less fun to choose.

I see Warrior as a class that should have good physical combat skills, but is held back by weak starting equipment and limited access to money/resources/support. This works great, starting out with a shitty iron sword and some paper-thin armor, despite having good stats, gives the class a built-in narrative - 'Acquire better gear'.

You'll often do this through engaging in risky early encounters, or through roleplay. Earning mammons to eventually get that big blade you wanted, fulfilling your power fantasy. (very fun!)

Whilst minor, the nerfs give me an anxiety that we will lose sight of 'the fun', losing the fun identity that comes with picking a class in favour of perfect balance. Who would want to roleplay as a mediocre fighter?


**Nobles**
I haven't experienced these properly yet, so subject to change. But, no, I don't see why Nobles would need 'tad bit better' stats, as that again would contrast against the fantasy of the role for sake of perfect balance.

The fantasy being that you are a Noble, a 'highborn' (and very rob-able) gentleman with the clothing and mammon to prove it. Your power in the round comes from your background, not your stats. You may look to join the Royal Court, or hire a bunch of thugs to Guard you. Stats are irrelevant here.


I can't speak for any of the other roles or changes. I just wanted to post this to highlight some worries I have about the direction of these decisions.

Please remember the fun. Please design for fun. <3

carbous commented:
ah that's good for the chainkini, the issue about amazon is that strenght is massive, but the bigger issue is the lack of con, it is really needed if they're expected not to use great armor, something like 2 con like the barb, it'd set them apart from other roles. As it stand in the way they were just made, they are worse warriors in about every ways.

ThePainkiller commented:
> ah that's good for the chainkini, the issue about amazon is that strenght is massive, but the bigger issue is the lack of con, it is really needed if they're expected not to use great armor, something like 2 con like the barb, it'd set them apart from other roles. As it stand in the way they were just made, they are worse warriors in about every ways.

Fair point, we could give Amazon an extra point in CON as they're supposed to be beefy and not wearing metal armors, it's just that she's considered agile, therefore the higher speed and climbing proficiencies, compromising raw strength. That might help equate both roles to be the same but of different gender

> Nerfing the Warriors base skills goes against the grain of the Classes power fantasy, making it a lot less fun to choose.

With no allegiances and full freedom to do what you want and even go away and never interact with the town, comes a price. Skills are able to be levelled much, MUCH more than they were before, between my PR #117 and this one, granting a lot more advancement for Pilgrims moreso than any other class. Pilgrims are assistants with pre-spawned items, and will be treated as such, with the possibility of growing more powerful at their discretion, rather than being better than a trained, paid, and well fed Town Guard. If you played any fantasy games and told me you can merk town guards at level one with starter equipment I'd ask what kinda games you've been playing. There's no _"power fantasy"_ here. You are **a** warrior, as are twenty other people that choose the role every week. We're not main characters of any story, no chosen ones of any prophecy. _Please remember that._ You start as a mediocre fighter, but at least you **ARE** a fighter, unlike that blacksmith travelling with just a hammer for his work, or a bard with their instrument and a simple dagger to defend themselves with. True joy is derived from the advancement, from the levelling, from the conquering of adversity. You will fight, you will survive, and you will triumph, you will buy better gear, you will kill greater beasts, and you will carve your own story for the week. There's no joy to be had from being an all wiping demigod clad in plate armor from minute zero that can one hit wolves, that fantasy's gonna get old real fast. That's also why rare classes are, by comparison, that much better. Sometimes even better than towner roles. That's why they cost triumphs to roll for.
We don't want things to be harder for people, we want to follow a set hierarchy of training in roles, and their respective freedom. I promise you won't even feel this as a nerf, at all, with all the other boons like this PR introducing athletics as "extra stamina", and SPECIALLY after me introducing the shield skill, making you parry 99% of hits as a Warrior from the get go, compared to your measly parry chance you had before with even Expert swords, or any other class that does NOT spawn with a shield. Test, and see, and you won't even feel the changes. Everything was made just **fairer** towards TOWN roles, and less Pilgrim steamrolling as it was before. A guard that is ill geared to fight a pilgrim is not something you pay a salary for, nor gear with weapons for, or waste time money and food to train only for a travelling dude to be better at you in every aspect. Less Skyrim, more Kingdom Come Deliverance. Without being overly punishing, Vulves and Goblins will still pose zero issue to you, and if you train hard enough, accrue money enough to buy upgrades, and gather a good party of comrades, you WILL tackle higher challenges, rather than taking the world on your own.

zeratino commented:
can you give swordmaster and lancer reading skills

Stutternov commented:
> can you give swordmaster and lancer reading skills

Added, gave them reading 2. No idea why they couldn't read before given they are a rare role plus they are supposed to be learned. 

> ah that's good for the chainkini, the issue about amazon is that strenght is massive, but the bigger issue is the lack of con, it is really needed if they're expected not to use great armor, something like 2 con like the barb, it'd set them apart from other roles. As it stand in the way they were just made, they are worse warriors in about every ways.
> 
> about the noble, the main issue is that they're hurt by the lack of both skills, stats and they've got no mammons, they seem to spawn with maybe max 70, which isn't great for a role based around riches. I suggest they get 200 mammons, it's a very good sum but it's not enough to ruin the economy or anything.

Added constitution to Amazon, and gave Nobles some (OK) starting weapons; i.e a bow for the female and a sword for the male. I also gave them a pouch of random value gold coins, basically rolling the same odds as the silver coins individually but instead of silver it's gold. So, should commonly give you way more bang for your buck.

Stutternov commented:
> **Warrior Nerfening, and Power Fantasy**
> 
> Nerfing the Warriors base skills goes against the grain of the Classes power fantasy, making it a lot less fun to choose.
> 
> I see Warrior as a class that should have good physical combat skills, but is held back by weak starting equipment and limited access to money/resources/support. This works great, starting out with a shitty iron sword and some paper-thin armor, despite having good stats, gives the class a built-in narrative - 'Acquire better gear'.
> 
> You'll often do this through engaging in risky early encounters, or through roleplay. Earning mammons to eventually get that big blade you wanted, fulfilling your power fantasy. (very fun!)
> 
> Whilst minor, the nerfs give me an anxiety that we will lose sight of 'the fun', losing the fun identity that comes with picking a class in favour of perfect balance. Who would want to roleplay as a mediocre fighter?

I will give my reason here - if you look, all I did was decrease some of their trainings by one, sometimes just removing the random chance and making it set.

This is an **infinite** spawn role, unlike sword master, who only spawns one. And, swordmaster _only_ gets 4 skill in sword. To put this in comparison with town garrison, just for an example of a combat class with limited job slots:

**Garrison Guard** get 3 polearms, 2 swords, 3 wrestling, 2 unarmed, 2 knives, 3 axes, 2 shield, no sword, and only 2 athletics. (Stats: 1 strength, 1 perception, 1 endurance, 1 constitution)
**Warrior** _used_ to get 1-2 crossbow, 3-4 polearms, 3-4 axes, 1-2 bow, 2 wrestling, 2 unarmed, 3-4 sword, 1-3 knives, 1-2 shield, 4 athletics,  _plus_ heavy armor training. (Stats: 2 strength, 1 endurance, 1 speed)

**Warrior** _now_ gets 1-2 crossbow, 2 polearm, 2 axe, 1-2 bow, 2 wrestling, 2 unarmed, 2-3 sword, 1-3 knives, 1-2 shield, 4 athletics, and kept its heavy armor training and stats have been left exactly the same.

Warrior still has its good stats, and the issue is - its skills are **higher**, starting, than even some rare roles, some better stats than even the Guard Captain, and on-par with the royal guard; roles that are very limited in slots. The issue is warriors were outright even better than any mercenary due to this.

I reduced the training since warrior is a 'jack of all trades, master of none', and skills actually matter less than stats, as you can simply level those - unlike stats, where you can't. If I gave warrior skills back, I would need to make them less of a jack of all trades to make them more fair. If it was not an infinite slot role, I don't think this would be an issue remotely. 

I don't think this really changes the gameplay loop much besides encouraging you to actually play the game more like an RPG and level your stats, which you can do naturally through being a warrior, and your greatest boon is still those stats (strength and speed are strong). This should give rare combat roles and specialized combat roles a slight edge while warrior still actually has some major bonuses; from its stats, to the VERY useful shield training, to being one of the only pilgrim classes that can actually wear full plate. (Something devs told me prior they wanted removed, but I think is fine since Warrior is a 'build-a-class' almost.)

Stutternov commented on code/__DEFINES/traits.dm:
```diff
@@ -266,7 +265,6 @@ Remember to update _globalvars/traits.dm if you're adding/removing/renaming trai
 #define TRAIT_PHOTOGRAPHER		"photographer"
 #define TRAIT_MUSICIAN			"musician"
 #define TRAIT_LIGHT_DRINKER		"light_drinker"
-#define TRAIT_EMPATH			"empath"
```
They are duplicates.

Stutternov replied:
They are defined TWICE in the same .dm file and it generates errors on compile.

Stutternov replied:
Example, line 13 on trait.dm file has Empath still there.

Same with anti-magic on line 36.
